oidc-api/src/main/java/uk/gov/di/authentication/oidc/helpers/RequestObjectToAuthRequestHelper.java

```diff
@@ -41,7 +42,19 @@ public static AuthenticationRequest transform(AuthenticationRequest authRequest)
                             .requestObject(authRequest.getRequestObject());
 
             if (Objects.nonNull(jwtClaimsSet.getClaim("vtr"))) {
-                builder.customParameter("vtr", (String) jwtClaimsSet.getClaim("vtr"));
+                if (jwtClaimsSet.getClaim("vtr") instanceof String vtr) {
```

Comment:
Could define `var vtrClaim = jwtClaimsSet.getClaim("vtr")` at the top of the method